### PR TITLE
Added Alison Meynert to contributors list

### PR DESCRIPTION
**Pull request by ameynert**

### CODE REVIEW

File: nf-core-contributors.yaml [hunk 1]
```diff
@@ -560,3 +560,17 @@ contributors:
       contact_github: jcurado-flomics
       location: [41.385251, 2.194007]
       twitter: flomicsbiotech
+      
+    - full_name: MRC Human Genetics Unit, MRC Institute of Genetics and Molecular Medicine, University of Edinburgh
+      short_name: MRCHGU
+      description: >
+        The MRC Human Genetics Unit discovers how changes in our DNA impact our lives. We combine the latest computational and experimental technologies to investigate how our genomes work to control the function of molecules, cells and tissues in people and populations.
```
Comment (ewels) [inline nf-core-contributors.yaml:567]:
```suggestion
        The MRC Human Genetics Unit discovers how changes in our DNA impact our lives.
        We combine the latest computational and experimental technologies to investigate
        how our genomes work to control the function of molecules, cells and tissues in people and populations.
```

File: University-of-Edinburgh-01.svg [hunk 2]
```diff
@@ -0,0 +1,1367 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (ewels) [inline University-of-Edinburgh-01.svg:1]: Please move this file to the `nf-co.re/public_html/assets/img/contributors-colour/` folder 👍

**Comment by ameynert** [comment]
Requested changes made, thanks!

**Comment by ewels** [comment]
Looking better! Though the white Edinburgh logo file appears to actually be the IARC WHO logo? 😄

**Comment by ameynert** [comment]
Let's try that again ... clearly I should not be on git after my bedtime! :)

**Comment by ewels** [comment]
Nice, getting there! However the logo now seems to be four logos stuck together. I understand the logic behind this, but the super wide shape won't really work on the website. The logos will be so small that you basically won't be able to see them.

I'm attaching a screenshot as I'm feeling guilty and want to prove that I'm not making this stuff up!

image


Please could you pick just one please?

Phil

**Comment by ameynert** [comment]
I hope that's sorted it now! Sorry about taking so much of your time on this. The multi logo thing has a massive political backstory, but graphics came through for me this morning with the single logo.

**Comment by ewels** [comment]
I can imagine 😅 

Sorry for being picky, and thanks for sorting it out! 👍

**Comment by ewels** [comment]
Ah so close! I didn't check the white version closely enough 😆 

image

No worries, I'll fix this now..

**Comment by ewels** [comment]
ok, thanks for merging @drpatelh - we're done! Looking good! 🎉 

image


image

**Pull request merged**